### ISSUE TITLE
Large amount of Unable to Connect to tcp://apps.vbresults.com:80. Error #110: Connection timed out errors in serv logs

### ISSUE THREAD

**Issue by Dannymh** [vbresults/mobile-apps-for-xenforo#43]
Over the past 24 hours have had a lot of the following error being generated

Unable to Connect to tcp://apps.vbresults.com:80. Error #110: Connection timed out

Zend_Http_Client_Adapter_Exception: Unable to Connect to tcp://apps.vbresults.com:80. Error #110: Connection timed out - library/Zend/Http/Client/Adapter/Socket.php:235

`#0 /var/www//library/Zend/Http/Client.php(973): Zend_Http_Client_Adapter_Socket->connect('apps.vbresults....', 80, false)
#1 /var/www//library/MobileInterface/PushAlert/Transport.php(42): Zend_Http_Client->request('POST')
#2 /var/www//library/MobileInterface/PushAlert/Transport.php(14): MobileInterface_PushAlert_Transport::send('register', Array)
#3 /var/www//library/MobileInterface/ControllerPublic/Account.php(5): MobileInterface_PushAlert_Transport::register(Object(Valhalla_LikesGiven_ControllerPublic_Account))
#4 /var/www//library/XenForo/FrontController.php(351): MobileInterface_ControllerPublic_Account->actionPushAlertRegister()
#5 /var/www//library/XenForo/FrontController.php(134): XenForo_FrontController->dispatch(Object(XenForo_RouteMatch))
#6 /var/www//index.php(13): XenForo_FrontController->run()
#7 {main}`

Other socket connections seem to be working, and if I curl on the command line it connects.

**Comment by Dannymh** [comment]
May be due to something server side, am trying to check with fsockopen and ensure ubuntu is allowing the external connections

**Comment by Dannymh** [comment]
Ok so I ran a test to my home IP address, to another site and to an IP Address fsocket times out going to apps.vbresults.com it always timesout. I also increased the time out on my test to 30 seconds with the same result.

I notice you are using cloudflare and if I do a curl I get a 301 redirect meessage so I wonder if that has something to do with it.

Most likely this is my server but I can connect to many other URL's from the same server via fsocket, So perhaps it is something blocking there. It is preventing push notifications from going out

**Comment by Dannymh** [comment]
I tried the connection from another server and that connects.

Is it possible that vbresults.com is blocking my servers IP address for some reason, hence why it times out from one server and not the other?

**Comment by Dannymh** [comment]
Ok I fixed the issue.

The issue is that I had ipv6 enabled on my server. Which means that because your site has ipv6 enabled for the URL, it will continue to try to use ipv6 to make connections to your site. However for somereason, perhaps due to configuration ipv6 connections for fsock seem to fail. I disabled ipv6 on my server and re-ran the connection and now I have success.

I am going to reconfigure ipv6 on my server again soon and make sure that I have configured my own DNS for ipv6 correctly for my domain
